include/nil/blueprint/blueprint/plonk/circuit.hpp

```diff
@@ -94,7 +94,7 @@ namespace nil {
             }
 
             crypto3::zk::snark::plonk_lookup_constraint<BlueprintFieldType>
-                add_lookup_constraint(std::vector<crypto3::math::non_linear_term<crypto3::zk::snark::plonk_variable<BlueprintFieldType>>> lookup_input, 
```

Comment:
This function isn't even used, it only appears in a commented out form in 
./include/nil/blueprint/components/hashes/sha256/plonk/sha256_process.hpp

Reply:
That's because for many components it is not used for now.